### PR TITLE
Add code to check genome section in MANIFEST before loading sample

### PR DESCRIPTION
**Pull request by naegelyd**
This includes tests to cover the different cases where the genome
section of the MANIFEST is invalid.

Signed-off-by: Don Naegely naegelyd@gmail.com

### CODE REVIEW

File: vdw/samples/pipeline/handlers.py [hunk 1]
```diff
@@ -51,26 +52,39 @@ def check_sample_section(manifest):
 def load_samples(manifest_path, database, **kwargs):
     manifest = ManifestReader(manifest_path)
 
-    # Ensure the sample is marked to be loaded..
+    # If a specific version of the genome is required then make sure the
+    # MANIFEST lists that version or abort the load process.
+    genome_version = getattr(settings, 'REQUIRED_GENOME_VERSION', None)
```
Comment (bruth) [inline vdw/samples/pipeline/handlers.py]: Prefix setting with `VDW_` to namespace it.

File: vdw/samples/pipeline/handlers.py [hunk 2]
```diff
@@ -51,26 +52,39 @@ def check_sample_section(manifest):
...
+    if genome_version:
+        genome_info = manifest.section('genome')
+
+        if not genome_info:
+            log.info('Genome version "{0}" is required but genome section is '
```
Comment (bruth) [inline vdw/samples/pipeline/handlers.py]: This and the below log message feel more like warnings.

Comment (naegelyd) [inline vdw/samples/pipeline/handlers.py]: I was following the convention already in place in that file, namely [this](https://github.com/cbmi/varify-data-warehouse/blob/8ba5d5e5a554bd67016af4778cb8efa8c307384e/vdw/samples/pipeline/handlers.py#L61-L66) and [this](https://github.com/cbmi/varify-data-warehouse/blob/8ba5d5e5a554bd67016af4778cb8efa8c307384e/vdw/samples/pipeline/handlers.py#L91-L95) but I agree they do seem more like warnings. I will update them.

Comment (bruth) [inline vdw/samples/pipeline/handlers.py]: Feel free to change the others to warnings as well.

**Pull request merged**